### PR TITLE
Dt new js quiz

### PR DESCRIPTION
**Pull request by kdylanthomas**
overwriting old JS quiz files with new quiz files for the "grow your tree" exercise

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
-console.log("str", str);
-console.log("arr", arr);
-console.log(document.getElementById("fruit-list").innerHTML);
 
+form.addEventListener("submit", submitTree);
 
+function submitTree(event) {
+	console.log("event", event);
+	event.preventDefault();
+	growYourTree(tree);
```
Comment (gregkorte) [inline quiz.js]: Take a close look at this object you are passing in. I'm not sure that as it's written it's necessary. Try console logging before the object gets passed in.

**Comment by gregkorte** [comment]
Dylan! Nice job on your quiz. Check out the code comment I sent you. Nice error checking. Send another PR once you find and correct it.

**Comment by gregkorte** [comment]
So it states in the instructions that the function must accept an object. Take another look, and see if you can solve the challenge.

**Comment by kdylanthomas** [comment]
You're right, oops! Okay, I am pretty stumped, but I went ahead and pushed my most recent commit with the tree parameter re-added. I'm going to keep looking at it, but I might wait to ask some in-person questions tomorrow depending on how far I get.

**Pull request closed without merging**